src/binaryen-c.h

```diff
@@ -103,6 +103,7 @@ typedef void* BinaryenFunctionTypeRef;
 // Add a new function type. This is thread-safe.
 // Note: name can be NULL, in which case we auto-generate a name
 BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
+BinaryenFunctionTypeRef BinaryenGetFunctionType(BinaryenModuleRef module, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
```

Comment:
We could add an API call to run a pass by name, yeah, right now it's pretty limited to optimize or not. Is that what you mean in the first paragraph?

Ok, I'll add that optimization.

I might be wrong, but to me it feels like such a getter could be implemented in JS, if we provided enough lower-level functionality, which feels more natural to me. Open to other opinions here though.